sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -418,9 +415,10 @@ static void appendCompileOptionsFromImage(std::string &CompileOpts,
     if (detail::SYCLConfig<detail::SYCL_RT_WARNING_LEVEL>::get() == 0)
       CompileOpts += " -disable-finalizer-msg";
   }
-  if (isDoubleGRFEsimdImage) {
-    assert(!CompileOpts.empty()); // -vc-codegen must be present
-    CompileOpts += " -doubleGRF";
+  if (isDoubleGRF) {
+    if (!CompileOpts.empty())
+      CompileOpts += " ";
+    CompileOpts += "-ze-opt-large-register-file";
```

Comment:
Can we end up passing this option to OpenCL or are we guaranteed that the image won't ever be considered by OpenCL?

Reply:
@kbobrovs, do you know the answer to this? Thanks

Reply:
Yes, I think this place is generic for SPIRV BEs. But according to @vmustya, Intel's OpenCL RT should support "-ze-opt-large-register-file".

Reply:
@steffenlarsen Hey, do you have any flags for this change? Hoping to get this merged soon since it's been in review for a while. Thanks

Reply:
I think that covered my concerns, thanks!